src/v2i-hub/CDASimAdapter/src/CDASimConnection.cpp

```diff
@@ -20,7 +20,13 @@ namespace CDASimAdapter{
     }
 
     bool CDASimConnection::connect() {
-        if (!carma_simulation_handshake(_simulation_ip, _infrastructure_id, _simulation_registration_port, _local_ip, _time_sync_port, _v2x_port, _location)) {
+        //Read local sensor file and populate the sensors JSON
+        populate_sensors_with_file(_sensor_json_file_path, _sensors_json_v);
+        if(_sensors_json_v.empty())
+        {
+            PLOG(logERROR) << "Sensors JSON is empty!" << std::endl;
```

Comment:
It is not an error for the Sensors configuration file to be empty. Handshakes with no sensors configured should still be valid

Reply:
Addressed